README.md

```diff
@@ -20,15 +20,16 @@ Finally it has built-in support for authentication, transport encryption and com
 - **Lime.Protocol**: Base protocol ([NuGet](https://www.nuget.org/Packages/Lime.Protocol))
 - **Lime.Transport.Tcp**: TCP transport implementation ([NuGet](https://www.nuget.org/Packages/Lime.Transport.Tcp))
 - **Lime.Transport.WebSocket**: WebSocket transport implementation ([NuGet](https://www.nuget.org/Packages/Lime.Transport.WebSocket))
-- **Lime.Transport.Http**: HTTP emulation layer (LIME over HTTP) ([NuGet](https://www.nuget.org/Packages/Lime.Transport.Http))
+- **Lime.Transport.AspNetCore**: Middleware for hosting Lime in the ASP.NET Core infrastructure ([NuGet](https://www.nuget.org/Packages/Lime.Transport.AspNetCore))
```

Comment:
I think is worthed to say this is an experimental feature at this time, @andrebires

Reply:
Done